lpot/adaptor/mxnet.yaml

```
@@ -125,6 +125,71 @@
     << : *common_patterns
 
 
+version:
+    name: '1.9.0'
+
+  precisions:
+    names: int8, fp32
+    valid_mixed_precisions: []
+
+  ops:
+    int8: ['_sg_mkldnn_conv', 'Convolution', '_sg_mkldnn_fully_connected', 'FullyConnected', '_sg_mkldnn_selfatt_qk', '_sg_mkldnn_selfatt_valatt']
+    fp32: ['*'] # '*' means all op types
+
+  capabilities:
+    'int8': {
+        '_sg_mkldnn_conv': {
+            'activation': {
+                'dtype': ['int8', 'fp32'],
```

Would it be possible to add "autoint8" to the possible values of "dtype" attribute, which would correspond to the automatic selection of dtype according to the calibration result of each tensor?
```suggestion
                'dtype': ['autoint8', 'fp32'],
```
This way we could describe more clearly what is actually happening in MXNet.